buildroot-external/board/pc/generic-x86-64/cmdline.txt

```diff
@@ -1 +1 @@
-console=tty1
+cmdline="console=tty1"
```

Comment:
If this is done in x86 with the console arg in cmdline.txt, maybe do the same for ARM64?

Reply:
Not sure if its really required, but we can add it if it helps :man_shrugging: 

Afaik, the ARM64 variant used the serial console. I'd like to keep that if possible. Does ARM64 has a graphical console? Enabling one `-device virtio-gpu-pci` did not give me a graphical console when I tried.

Reply:
I mean, the QEMU ARM64 model does not seem to have graphics at all from what I could find yeah.

Reply:
Now with `CONFIG_DRM_VIRTIO_GPU=m` it does seem to support graphics.

Reply:
Turns out, to get boot up console we actually have to specify `console=tty1`. I'll adjust that for `generic-aarch64`.